### PR TITLE
remove unused underlines

### PR DESCRIPTION
**Pull request by vivianjeng**

### CODE REVIEW

File: packages/contracts/contracts/libraries/VerifySignature.sol [hunk 1]
```diff
@@ -5,10 +5,10 @@ import '@openzeppelin/contracts/utils/cryptography/ECDSA.sol';
 // verify signature use for relayer
 // NOTE: This method not safe, contract may attack by signature replay.
 contract VerifySignature {
-    /*
+    /**
      * Verify if the signer has a valid signature as claimed
      * @param signer The address of user who wants to perform an action
-     * @param siganture The signature signed by the signer
+     * @param signature The signature signed by the signer
```
Comment (vuvoth) [inline packages/contracts/contracts/libraries/VerifySignature.sol:11]: Thanks!

File: packages/core/src/utils.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ const genEpochKey = (
     identityNullifier: SnarkBigInt,
     epoch: number,
     nonce: number,
-    _epochTreeDepth: number = EPOCH_TREE_DEPTH
+    epochTreeDepth: number = EPOCH_TREE_DEPTH
```
Comment (vuvoth) [inline packages/core/src/utils.ts:77]:
```suggestion
    epochTreeDepth = EPOCH_TREE_DEPTH
```

File: packages/contracts/test/utils.ts [hunk 3]
```diff
@@ -146,16 +145,16 @@ const defaultGSTLeaf = (treeDepth: number): BigInt => {
 }
 
 const genNewUserStateTree = async (
-    _userStateTreeDepth: number = USER_STATE_TREE_DEPTH
+    userStateTreeDepth: number = USER_STATE_TREE_DEPTH
```
Comment (vuvoth) [inline packages/contracts/test/utils.ts:148]:
```suggestion
    userStateTreeDepth = USER_STATE_TREE_DEPTH
```

File: packages/contracts/test/utils.ts [hunk 4]
```diff
@@ -146,16 +145,16 @@ const defaultGSTLeaf = (treeDepth: number): BigInt => {
...
-    return genNewSMT(_userStateTreeDepth, defaultUserStateLeaf)
+    return genNewSMT(userStateTreeDepth, defaultUserStateLeaf)
 }
 
 const genEpochKey = (
     identityNullifier: crypto.SnarkBigInt,
     epoch: BigNumberish,
     nonce: BigNumberish,
-    _epochTreeDepth: number = EPOCH_TREE_DEPTH
+    epochTreeDepth: number = EPOCH_TREE_DEPTH
```
Comment (vuvoth) [inline packages/contracts/test/utils.ts:157]:
```suggestion
    epochTreeDepth = EPOCH_TREE_DEPTH
```

File: packages/contracts/test/utils.ts [hunk 5]
```diff
@@ -117,10 +116,10 @@ const genNewSMT = async (
 }
 
 const genNewEpochTree = async (
-    _epochTreeDepth: number = EPOCH_TREE_DEPTH
+    epochTreeDepth: number = EPOCH_TREE_DEPTH
```
Comment (vuvoth) [inline packages/contracts/test/utils.ts:119]:
```suggestion
    epochTreeDepth = EPOCH_TREE_DEPTH
```

**Review by vuvoth — APPROVED** [review]
Changes look good to me!

**Review by vimwitch — APPROVED** [review]

**Pull request merged**